### PR TITLE
Update Frameworks.md

### PR DESCRIPTION
**Pull request by antimatter96**
Currently adding Expressjs. Hope to complete it by 2016/10/20

### CODE REVIEW

**Comment by antimatter96** [comment]
#10

File: Frameworks.md [hunk 1]
```diff
@@ -115,3 +115,43 @@ include the Django Rest Framework (DRF), Channels, Elasticsearch and Celery.
...
+the database by constructing SQL queries.
+
+The views defines the overall presentation of the data. This defines how the front end will look if a user receives
+the response from the controller. Templates helps in reducing code repition as they allow importing common elements from other
+templates. Templates have their own syntax that introduces logic and variables within the typical HTML markup. This makes it possible
+to access the content of the context. For instance, one can loop through all the data of a query while applying the
+correct styling. Lastly, templates can import from a base template, which allow common elements, such as a navbar, to be
+written once and plugged where needed.
+
+Django also includes more advance features such as an administrative interface, user authentication and a caching
```
Comment (mbad0la) [inline Frameworks.md]: Why is there content on Django for the ExpressJS section?

**Review by mbad0la — CHANGES_REQUESTED** [review]

File: Frameworks.md (hunk 1 above)
Comment (antimatter96) [reply Frameworks.md]: Was using as template.

Comment (mbad0la) [reply Frameworks.md]: By when will this PR be ready?

**Comment by antimatter96** [comment]
Was using as template. Need changes

**Comment by antimatter96** [comment]
@mbad0la done

**Comment by mbad0la** [comment]
Will check it out as soon as possible!

Mayank Badola
University of Delhi - '17
B.E. Computer Engineering
Netaji Subhas Institute of Technology

**Review by mbad0la — APPROVED** [review]

**Pull request merged**